api/src/groups/api.py

```diff
@@ -561,16 +562,22 @@ def update_member(group_id, person_id):
         # check if the new person exists
         if new_person_id and not db.session.query(Person).filter_by(id=new_person_id).first():
             return logged_response(f"Person with person_id #{new_person_id} does not exist", 404)
+    new_active = valid_attributes.get('active')
+
+    if new_active is False and member.active is True: # if will deactivate person
+        create_member_history(member, is_join=False) # create a leave entry on the original group
+
+    if member.active and new_active is not False and new_group_id and new_group_id != member.group_id: # if will move the active member
+        create_member_history(member, is_join=False) # create a leave entry on the original group
+        create_member_history(member, is_join=True, group_id=new_group_id) # create a join entry on the new group
 
-    if ((valid_attributes.get('active') is False and member.active is True) # if will deactivate person
-            or (new_group_id and new_group_id != member.group_id) # if will move person to another group
-            # or (new_person_id and new_person_id != member.person_id) # if will replace the current member with another person
-            ):
-        create_member_history(member)
+    if new_active is True and member.active is False: # if will activate person
```

Comment:
`is True` is not necessary. I'd suggest searching for instances of `is True` and similar Boolean phrases and simplifying throughout.